app/javascript/application.js

```diff
@@ -106,4 +102,19 @@ $(document).ready(function () {
   show('.feedback', 'form-reply-', '.form-reply');
   show('.edit-reply', 'form-reply-edit-', '.form-reply-edit');
   show('.edit-comment', 'form-comment-', '.form-edit');
-  
\ No newline at end of file
+  
+  $(document).ready(function () {
+    var commentTextarea = $('textarea');
```

Comment:
Please check when to use let, var, const.
Normally you should only use let or const.